practicals/GNN_for_recommendations.ipynb

```diff
@@ -0,0 +1,1284 @@
+{
```

Comment:
<p>Reminder: everything that needs downloading and importing should be right at the start of the practical and clearly marked, so that attendees can download what is needed before attending the conference</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2024/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Does this mean we need to put at the top ? i think then we need to do the same for first section of the tutorial also. </p>

Reply:
<p>also currently we only install Jraph.. do you recommend we setup and installation for all others including libraries like pandas etc?</p>

Reply:
<p>Yes, it ideally needs to be near the top. Or there should be clear instructions near the top referring them to the cell.</p><p> </p><p>I.m.o you should include all the imports near the top, so that attendees can run the cell before coming and it will crash if they haven't installed all the right libraries.</p>

Reply:
I think this is done already and resolved.. thanks :)